Makefile.common

```diff
@@ -105,7 +105,7 @@ BYTERUN:=$(addprefix kernel/byterun/, \
 CORECMA:=lib/clib.cma lib/lib.cma kernel/kernel.cma intf/intf.cma library/library.cma \
         engine/engine.cma pretyping/pretyping.cma interp/interp.cma proofs/proofs.cma \
         parsing/parsing.cma printing/printing.cma tactics/tactics.cma vernac/vernac.cma \
-	stm/stm.cma toplevel/toplevel.cma parsing/highparsing.cma API/API.cma
+	parsing/highparsing.cma stm/stm.cma toplevel/toplevel.cma API/API.cma
```

Comment:
Sorry Paul, I am not sure I follow what you mean here... in the old situation you wouldn't be able to use `stm.cma` at all without also linking the toplevel and highparsing. In fact, hihgparsing is gone in #939